test/role.test.js

```diff
@@ -511,6 +546,24 @@ describe('role model', function() {
       });
     });
 
+    it('should resolve user by id using custom user principalType', function(done) {
+      ACL.resolvePrincipal('User', user.id, function(err, u) {
+        if (err) return done(err);
+
+        expect(u.id).to.eql(user.id);
+
+        done();
+      });
+    });
+
```

Comment:
testing here if using a user model name as principalType enables to resolve ACL's principal